avm/res/container-service/managed-cluster/main.bicep

```
@@ -352,6 +352,9 @@ param httpProxyConfig object?
 @description('Optional. Identities associated with the cluster.')
 param identityProfile object?
 
+@description('Enables Kubernetes Event-driven Autoscaling (KEDA)')
```

```suggestion
@description('Optional. Enables Kubernetes Event-driven Autoscaling (KEDA)')
```

Not, requires a re-run of the `Set-AVMModule` script

---

@PixelRobots this should have failed the static tests which I can see did not run in your fork. Is there a particular reason? I'd recommend to always run them to wrap up a PR. Otherwise the change may fail in the upstream repository and the module feature would actually not be published 👀

---

Just fixing it now. I had to run the pester tests locally to see the actual failure. the github action did not show it.